### PR TITLE
emqx v5.1.0

### PR DESCRIPTION
**Pull request by id**

### CODE REVIEW

File: library/emqx [hunk 1]
```diff
@@ -8,8 +8,14 @@ GitCommit: 129bff4ea692ede8157aa3165a8c278bf982f47d
 Directory: 4.4
```
Comment (tianon) [inline library/emqx:8]: I don't know if we've talked about it with you recently, but it seems like 4.4.x might be EOL now? :eyes:

> Removing tags here will remove them from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding them on the official build servers), but the tags will still be available to users who want them.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Comment (id) [reply library/emqx:8]: @tianon not yet, but soon. 4.4.0 was released on February 18 2022, EOL is in 18 months == August 18 2023 if my math is correct.

Comment (id) [reply library/emqx:8]: We expect one more 4.4.x release, after that we will remove this tag.

Comment (tianon) [reply library/emqx:8]: Ah, makes sense; not a problem at all, thanks for double checking and sharing your plan! :heart:

**Pull request merged**